website/views.py

```diff
@@ -34,13 +35,19 @@ def get_blocks():
 def dispatcher(request, permalink=''):
 
     blocks = get_blocks()
+    rssfeed = get_rss()
     banner = Banner.objects.filter(visible=1)
     context = {
         'navs': blocks['navs'],
         'sidebar': blocks['sidebar'],
         'footer': blocks['footer'],
-        'permalink': permalink,
+        'permalink': permalink
     }
+    if not rssfeed:
```

Comment:
This can be in the ```get_rss``` function itself. So the function returns this default value in case of empty rss.
Will be better if the logic for a particular objective is at one place.

Reply:
I have updated.